dicom-archive/profileTemplate.pl

```diff
@@ -49,6 +49,12 @@ sub getSubjectIDs {
         $subjectID{'visitLabel'} = NeuroDB::MRI::my_trim($patientName);
         $subjectID{'isPhantom'} = 1;
 
+        # If config setting 'useProject' and 'createCandidates' are true
+        # $subjectID{'ProjectID'} must be set.
+        $subjectID{'ProjectID'} = 1;
+
```

Comment:
what about the SubprojectID?

Reply:
The more I think about it, the more I feel like not providing a default for ProjectID and SubprojectID: I'd rather indicate with an appropriate comment that these must be set when `createCandidate`/`createVisitLabel` are turned on. There is not even a default project defined in `0000-00-00-schema.sql` so `$subjectID{'ProjectID'} = 1` does not even make much sense when I think about it...Let me know what you think.

Reply:
I looked at the default schema and it looks like this [line](https://github.com/aces/Loris/blob/12a235e511b810d35c39475db1b48f04fa8e85c9/SQL/0000-00-00-schema.sql#L13) would create a ProjectID = 1 so I guess there would be a default ProjectID = 1 anyway?